### PR TITLE
Unit test support in non-English countries.

### PR DESCRIPTION
**Pull request by kurobon-jp**
Hello.
Some of unit tests don't work in my area.
capture

So we added the function to set the default culture.
However, this may not be the best as not all test classes inherit AbstractTestFixture.
Are there any other good ideas?

### CODE REVIEW

**Comment by filipw** [comment]
I think if you added it into the static constructor, it should also work https://github.com/OmniSharp/omnisharp-roslyn/blob/master/tests/TestUtility/SharedOmniSharpHostFixture.cs

**Comment by kurobon-jp** [comment]
Thank you for the advice.

File: tests/OmniSharp.Script.Tests/WorkspaceInformationTests.cs [hunk 1]
```diff
@@ -77,35 +77,44 @@ public async Task SingleCsiScriptWithCustomRspNamespacesAndReferences()
         public async Task CsiScriptWithFileCreatedAfterStartingServer()
         {
             using (var testProject = TestAssets.Instance.GetTestScript("EmptyScript"))
-            using (var host = CreateOmniSharpHost(testProject.Directory))
             {
-                var workspaceInfo = await GetWorkspaceInfoAsync(host);
-                Assert.Empty(workspaceInfo.Projects);
+                using (var host = CreateOmniSharpHost(testProject.Directory))
```
Comment (filipw) [inline tests/OmniSharp.Script.Tests/WorkspaceInformationTests.cs]: why is this test changed?

Comment (filipw) [reply tests/OmniSharp.Script.Tests/WorkspaceInformationTests.cs]: it actually changes the intent of the test, the intent was:
1) initialize workspace to some directory
2) verify no script files are there
3) create a file *after* the host is running (now with the change, the file is created *before* the host is running)
4) verify that the script file was picked up by a running host

Comment (kurobon-jp) [reply tests/OmniSharp.Script.Tests/WorkspaceInformationTests.cs]: In my environment, the original CsiScriptWithFileCreatedAfterStartingServer did not pass.
capture1
capture2

Comment (filipw) [reply tests/OmniSharp.Script.Tests/WorkspaceInformationTests.cs]: thanks for finding this. see https://github.com/OmniSharp/omnisharp-roslyn/pull/1570
once that is merged, you can rebase on top of latest master and everything should be fine without any need to change this test anymore.

Comment (kurobon-jp) [reply tests/OmniSharp.Script.Tests/WorkspaceInformationTests.cs]: Thank you @filipw.

**Pull request merged**